code/modules/mafia/controller.dm

```diff
@@ -551,7 +551,7 @@
 		if(player_client)
 			player_client.prefs.copy_to(H)
 			if(H.dna.species.outfit_important_for_life) //plasmamen
-				H.set_species(/datum/species/human)
+				H.set_species(/datum/species/plasmaman/stable)//Admin only plasmamen that don't breath or burn
```

Comment:
This assumes only plasmamen will be `outfit_important_for_life`. Replace `outfit_important_for_life` with instead of a boolean, a link to the "stable" subtype.

Reply:
Changed how this work so I will resolve this.